Call to staticmethod in another module

### Acton Version

0.11.7.20221026.12.55.38

### Steps to Reproduce

With mod1.act
```
from mod2 import *

def foo():
    Bar.test()
```
and mod2.act
```
class Bar:
    @staticmethod
    def test():
        pass
```
compile with `actonc build`

### Expected Behavior

Compilation success

### Actual Behavior

```
Building project in /home/ddjoh/workspaces/act-test-1
  Compiling mod2.act for release
   Already up to date, in    0.000 s
  Compiling mod1.act for release
ERROR: internal compiler error: compilation of generated C code failed
NOTE: this is likely a bug in actonc, please report this at:
NOTE: https://github.com/actonlang/acton/issues/new?template=ice.md
NOTE: acton 0.11.7.20221026.12.55.38 compiled by ghc 8.10 on linux x86_64
NOTE: cc: cc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
cc stdout:

cc stderr:
act-test-1/out/types/mod1.c: In function ‘mod1$$foo’:
act-test-1/out/types/mod1.c:3:6: error: too few arguments to function ‘(struct $NoneType * (*)(struct mod2$$Bar *))mod2$$Bar$methods.test’
    3 |     (($NoneType (*) (mod2$$Bar))mod2$$Bar$methods.test)();
      |     ~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```

--- a/compiler/Acton/Converter.hs
+++ b/compiler/Acton/Converter.hs
@@ -271,8 +271,8 @@ fromTEnv (_ : te)                       = fromTEnv te
 fromTEnv []                             = []
 
 convClassTEnv env q0 te                 = [ (n, conv i) | (n,i) <- te ]
-  where conv (NSig sc dec)              = NSig (convS sc) NoDec
-        conv (NDef sc dec)              = NDef (convS sc) NoDec
+  where conv (NSig sc dec)              = NSig (convS sc) dec
+        conv (NDef sc dec)              = NDef (convS sc) dec
         conv i                          = i
         convS (TSchema l q t)           = TSchema l (noqual env q) (convT q t)
         convT q (TFun l x p k t)        = TFun l x (qualWRow env (q0++q) p) k t

